src/winetricks

```diff
@@ -12413,6 +12413,84 @@ load_pngfilt()
 
 #----------------------------------------------------------------
 
+w_metadata powershell_core apps \
+    title="PowerShell Core" \
+    publisher="Microsoft" \
+    year="2024" \
+    media="download" \
+    file1="PowerShell-7.2.21-win-x86.msi" \
+    file2="PowerShell-7.2.21-win-x64.msi"
+
+# Function to download and install PowerShell Core
+load_powershell_core()
+{
+    # Uncomment below and remove win32 download elif and file when PowerShell Core v7.2 LTS goes EOL
+    #w_package_unsupported_win32
+
+    # Download PowerShell Core 7.2.x MSI (Latest LTS Version to support win32)
+    # https://github.com/PowerShell/PowerShell/releases/v7.2.21
+    if [ "${W_ARCH}" = "win64" ]; then
+        w_download "https://github.com/PowerShell/PowerShell/releases/download/v7.2.21/PowerShell-7.2.21-win-x64.msi" 407640b11c89d66ec7892229e68b1d74b26f0e820b52da268c67fd166c2b46ad
+        msi="${file2}"
```

Comment:
Almost there :).

This causes a shellcheck error (see the github actions).

Shellcheck doesn't know that the metadata is sourced before the function is called. You can either disable the warning, or remove the variables and hardcode it, up to you.

Reply:
fixed with https://github.com/Winetricks/winetricks/pull/2211/commits/768e913101d419be762a93c142e2814808179d8d